### DISCUSSION TITLE
Support for the E32S3RGB43 ?

### DISCUSSION THREAD

**Discussion by Smanar** [Bodmer/TFT_eSPI#3141] [Compatible displays and setup files]
Hello, do you know if the lib can support this display, even with some work, only for display, I know the touch will not work.

https://wiki.makerfabs.com/ESP32_S3_Parallel_4.3_TFT_with_Touch.html

LCD interface: RGB 565
LCD drivers : Unknown, similar to ST7282, HX8664/HX8264 from a PDF
I can't see if it's a 8 or 16bits parallel, 16 I think.

**Comment by Bodmer** [comment]
Those LCD drivers are not supported but maybe one of the other drivers will work. Without detailed information and experiemntation it will be tricky to get working.

The library supports 8 bit parallel only with the ESP32 S3.

**Reply by Bodmer** [reply]
It looks like it is 16 bit with all display drive refresh done by the ESP32 S3. Since that appears to be the case then the answer is no, the library does not support that display.

**Reply by Smanar** [reply]
That's what I feared, I know somes others libs can support it, but hard to find clear informations about this device.
BTW thx for information, I will give up with this device for the moment.
